README.md

```diff
@@ -81,11 +81,25 @@ check = plugnpy.Check()
 To add metrics to this check, simply use the add_metric() method of your Check object. This takes in arguments to add a Metric object to an internal array. It is these Metric objects that are called upon to create the final output.
```

Comment:
Change all code block sections for setup to be consistent - inc. the conda ones?